powersimdata/network/helpers.py

```diff
@@ -57,7 +57,12 @@ def interconnect_to_name(interconnect, model="hifld"):
     :param str model: the grid model.
     :return: (*str*): name of grid model.
     """
-    return "_".join(sorted(check_and_format_interconnect(interconnect, model=model)))
+    return "_".join(
+        sorted(
+            i.replace(" ", "")
```

Comment:
It allows the user to do instantiate the model with `Great Britain` or `Continental Europe` for an interconnect. We will be able to do:
```
from powersimdata import Scenario

scenario = Scenario()

scenario.set_grid(grid_model="europe_tub", interconnect="Continental Europe")
```

Reply:
I see. Then the user can also do "C o n t          in e  n t a l    Euro p  e", which is okay I think.

Reply:
It should work! I will add some tests.

Reply:
In fact *Continental Europe* (or any other interconnect with a space in it) could not be passed because the `check_and_format_interconnect` function in the same module would raise an error if the interconnect is not defined in the `model2interconnect` dictionary in the `powersimdata.network.constants.model` module.

I have updated the `check_and_format_interconnect` function to be able to pass both *Continental Europe* and *Continental Europe* and added some tests.